### PR TITLE
Stock/7 custom popup

### PR DESCRIPTION
**Pull request by pawellysy**

### CODE REVIEW

File: highcharts-api/highcharts-stock/7-custom-popup/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Stock exercise 7
+
+Create a custom popup for annotation module. When selecting annotation (clicking on it) the custom popup should show.
+The modal window should show in the middle of the chart, with a simple form. There should be one input field, which allows the user to change the color of the annotation, and the submit button. When the submit button is clicked, the custom popup should dissapear, and the options should be applied to the annotation. There should also be a close button to hide the popup with no effect. For reference, see the gif below.
```
Comment (Vazonik) [inline highcharts-api/highcharts-stock/7-custom-popup/README.md:4]:
```suggestion
Create a custom popup for the annotation module. When selecting annotation (clicking on it) the custom popup should show.
The modal window should show in the middle of the chart, with a simple form. There should be one input field, which allows the user to change the color of the annotation, and the submit button. When the submit button is clicked, the custom popup should disappear, and the options should be applied to the annotation. There should also be a close button to hide the popup with no effect. For reference, see the gif below.
```

File: highcharts-api/highcharts-stock/7-custom-popup/main.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      },
+      chartOptions: {
+        rangeSelector: {
+          inputEnabled: false
+        }
+      }
+    }]
+  }
+});
+});
```
Comment (Vazonik) [inline highcharts-api/highcharts-stock/7-custom-popup/main.js:113]: Something wrong happened with the indentations. Would be good to tidy it up.

**Review by Vazonik — CHANGES_REQUESTED** [review]
Nice task, requested some small fixes.

**Review by mfilipiec — APPROVED** [review]
LGTM

**Review by Vazonik — APPROVED** [review]
Thanks!

**Pull request merged**